crates/containerd-shim-wasmtime/src/instance.rs

```diff
@@ -243,13 +244,35 @@ impl Wasi {
         let stdout = maybe_open_stdio(stdout).context("could not open stdout")?;
         let stderr = maybe_open_stdio(stderr).context("could not open stderr")?;
 
+        let wasmtime_executor = Box::new(WasmtimeExecutor {
+            engine,
+            stdin,
+            stdout,
+            stderr,
+        });
+        let default_executor = Box::<DefaultExecutor>::default();
+
+        if let Some(stdin) = stdin {
```

Comment:
**question:** Are these needed just for testing?

Reply:
These are needed by the default container runtime to redirect stdio. I will update the code with comments.